tensorflow/c/eager/BUILD

```diff
@@ -256,6 +256,73 @@ tf_cuda_cc_test(
     ],
 )
 
+cc_library(
```

Comment:
The mobile build is failing because it is trying to include `mnist_gradients_util.*`. Could you rename these files to `mnist_gradients_testutil.h` and `mnist_gradients_testutil.cc`. That should exclude them from the mobile [srcs](https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/c/eager/BUILD;l=878;drc=ee95d88c4eb92311a8c57a8f78378235e1909d08).